meldebot/mel/flute.py

```diff
@@ -29,8 +29,7 @@ def get_random_flute():
         "3jW4feI2q_I",  # Harry Potter
         "SAAFA5T_GBE",  # Attack on Titan
     ]
-    vnum = randint(0, 3)
-    return [URLS[vnum]]
+    return choice(URLS)
```

Comment:
You are returning a different type. I think you should just fix `search_video_url` to expect the video key only